src/jira_client.ts

```diff
@@ -9,11 +9,10 @@ import {
 } from "fp-ts/lib/TaskEither";
 import * as t from "io-ts";
 import { readableReport } from "italia-ts-commons/lib/reporters";
-import { NonEmptyString } from "italia-ts-commons/lib/strings";
+import { EmailString, NonEmptyString } from "italia-ts-commons/lib/strings";
 import nodeFetch from "node-fetch";
 import { ServiceId } from "../generated/api/ServiceId";
 
-export const JIRA_SERVICE_TAG_PREFIX = "devportal-service-";
```

Comment:
Maybe is better keep hardcoded this prefix because is used not only in creation but even on reading. A wrong configuration could generate data inconsistency in Jira, creating card that should not be created.
I think that this service should be considered as a [model ](https://github.com/pagopa/io-functions-commons/blob/master/src/models/profile.ts#L26)so static configuration should be a better option.

Reply:
Fix with de55351